app.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const router = express.Router();
+const bodyParser = require("body-parser");
+const app = express();
+app.use("/", router);
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
```

Comment:
> In Express >= 4.16, body parser has been re-added under the methods express.json(). The replacement for the above question is
> 
> ```js
> const express = require("express");
> const app = express();
> app.use(express.json());
> app.use(express.urlencoded({ extended: true }));```

You can omit installing the `body-parser` package now. Refer to [this](https://stackoverflow.com/questions/55558402/what-is-the-meaning-of-bodyparser-urlencoded-extended-true-and-bodypar) thread for more information.